### PR TITLE
Include Pull action, update test.yml

### PR DESCRIPTION
**Pull request by chrisdchow**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -44,6 +44,19 @@ jobs:
...
     steps:
       - uses: actions/checkout@v2
-      - uses: ./
+      - name: Test Push
+        uses: ./
         with:
-          milliseconds: 1000
+          action: push
+          api-token: ${{ secrets.LOKALISE_API_TOKEN }}
+          project-id: 37600957607f701d17dfc3.52981513
```
Comment (abatilo) [inline .github/workflows/test.yml:52]: Would it make sense to actually make these a secret value too? Does it matter that much?

**Review by abatilo — COMMENTED** [review]
Quick question

File: .github/workflows/test.yml (hunk 1 above)
Comment (abatilo) [reply .github/workflows/test.yml:52]: Let's do that if you don't mind.  I just want to be extra cautious

Comment (chrisdchow) [reply .github/workflows/test.yml]: Done

**Review by abatilo — APPROVED** [review]

**Pull request merged**